pyproject.toml

```diff
@@ -37,13 +37,11 @@ exclude = '''
       aiobotocore
       | aiohttp
       | aiopg
-      | algoliasearch
       | boto
       | botocore
       | bottle
       | cassandra
       | celery
-      | consul
       | dbapi
       | dogpile_cache
       | falcon
```

Comment:
it looks like sqlite3, pymysql, dogpile, mysql and mongoengine were also formatted and can be removed from this list